data/remnant missions.txt

```diff
@@ -495,3 +560,703 @@ mission "Remnant: Technology Available"
+			`	"This section of our warehouse is the salvage area. Everything in here we have we collected in sufficient quantities that it is available for general use."`
+			`	She nods at a rack of Korath Heat Shunts while she explains, then turns back to you.`
+			`	"Of course the really advanced and rare stuff is still going to the labs first, and won't be available until we have learned all we can from it." She looks around proudly. "We don't have the industrial capacity or the numbers of researchers to keep up with the Alphas, the Korath, or whoever else might be out there. But we are very good at figuring out what others have built and learning from it.`
+			`	"Any salvage or unrecognized technology gets scanned in our quarantine area for threats, then looked over to figure out if there is anything new in it. If there is something new, it is shipped to a lab for our researchers to dissect. Anything we have already figured out ends up in here, where a select few captains are welcome to use it as they see fit."`
+			choice
+				`	"Threats?"`
+					goto threats
+				`	"Interesting."`
+			`	"Yes, it is, isn't it? So many new things out there to discover and learn about.`
+				goto end
+			label threats
+			`	"Yes, sometimes salvage that is brought in is dangerous. System Cores come with fleets of micro-bots, for instance. If they aren't de-activated properly, they keep trying to rebuild their surroundings in the form of the last ship they were configured to serve. And while it hasn't happened yet, we are worried that someday we might run into someone who builds traps into their equipment, or uses nanobots that have run amok. We haven't seen any of that yet, but our science-fiction stories are filled with tales of nanotech apocalypses and things like that.`
+			label end
+			`	"Well, I need to get back to work. That last battle put us behind schedule on our repairs. Have a good day!" With a final flourish that you think is the equivalent of an exclamation mark, Tali strides out of the outfitters, heading towards another landing bay.`
+
+event "Remnant Salvage Available"
+	outfitter "Remnant"
+		"Generator (Furnace Class)"
+		"Small Heat Shunt"
+		"Systems Core (Small)"
+		"Thruster (Asteroid Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Asteroid Class)"
+		"Steering (Planetary Class)"
+		"Korath Fire-Lance"
+
+
+
+mission "Remnant: Expanded Horizons Quarg 1"
+	name "Visiting the Quarg"
+	description "Take Remnant researchers to <destination> to see some Quarg."
+	minor
+	source
+		government "Remnant"
+	passengers 3
+	destination "Wayfarer"
+	to offer
+		has "Remnant: Learn Sign 3: done"
+		has "Remnant: Salvage 1: done"
+		has "First Contact: Quarg: offered"
+		random < 40
+	on offer
+		conversation
+			`As you relax in the cafeteria on <planet> you are approached by a young Remnant. Judging by the hand-held sensors and tools hanging off her belt, you suppose she is a researcher or an engineer.`
```

Comment:
`<planet>` is a text replacement for the destination planet. You want `<origin>` here.